redhat_upgrade_tool/commandline.py

```diff
@@ -85,9 +85,9 @@ def parse_args(gui=False):
         help=_('device or mountpoint. default: check mounted devices'))
     req.add_option('--iso', type="isofile",
         help=_('installation image file'))
-    # Translators: This is for '--network [VERSION]' in --help output
     req.add_option('--network', metavar=_('VERSION'), type="VERSION",
-        help=_('online repos matching VERSION (a number or "rawhide")'))
+        help=_('online repos. \'RELEASEVER\' will be used to replace'
+               ' &releasever variable should it occur in any repo URL.'))
```

Comment:
Should be *ampersand* really here ?

Reply:
Correct, it should be dollar sign. Thanks.